esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
+    full: 'Chrome',
+      short: 'CH 53',
+      note_id: 'experimental-flag',
+      unstable: true,
+  },
+  chrome54: {
+    full: 'Chrome',
+      short: 'CH 54',
```

Comment:
I can fix identation here if you want but I think it's better to create new PR. I can create it after this one will get merged